### PR TITLE
Building a Tiny JS World (pre-OOP) - done

### PR DESCRIPTION
**Pull request by evgenii-del**
_Complete the form below and remove this line._

### Building a Tiny JS World (pre-OOP) - practice

[Demo](http://careful-bean.surge.sh/) |
[Code base](https://github.com/evgenii-del/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by madmaxWMFU** [comment]
Why are you add PR which consist of two tasks?

**Comment by evgenii-del** [comment]
@madmaxWMFU I deleted the second task

File: submissions/evgenii-del/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+const dog = {
+    species: 'dog',
+    name: 'Bob',
+    gender: 'male',
+    legs: 4,
+    hands: 0,
+    saying: 'woof-woof!'
+};
+
+[man, woman, cat, dog].forEach(item => print([item.species, item.name, item.gender, item.legs, item.hands, item.saying].join(';')));
```
Comment (madmaxWMFU) [inline submissions/evgenii-del/tiny-js-world/index.js:47]: Write a function that returns a string with elements of an object.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]

**Review by madmaxWMFU — APPROVED** [review]

**Review by MaxKalashnyk — APPROVED** [review]
LGTM!

**Pull request merged**